src/DistributedArrays.jl

```diff
@@ -861,8 +861,8 @@ end
 (-)(x::Number, A::DArray) = x .- A
 
 map_localparts(f::Callable, d::DArray) = DArray(i->f(localpart(d)), d)
-map_localparts(f::Callable, d1::DArray, d2::DArray) = DArray(d1) do I
-    f(localpart(d1), localpart(d2))
+map_localparts(f::Callable, d::DArray, ds::DArray...) = DArray(d) do I
+    f(localpart(d), map(localpart, ds)...)
```

Comment:
You are right :) Didn't see that.

On Wed, 3 Aug 2016, 22:29 Amit Murthy, <notifications@github.com> wrote:

> In src/DistributedArrays.jl
> <https://github.com/JuliaParallel/DistributedArrays.jl/pull/94#discussion_r73454180>
> :
>
> > @@ -861,8 +861,8 @@ end
> >  (-)(x::Number, A::DArray) = x .- A
> >
> >  map_localparts(f::Callable, d::DArray) = DArray(i->f(localpart(d)), d)
> > -map_localparts(f::Callable, d1::DArray, d2::DArray) = DArray(d1) do I
> > -    f(localpart(d1), localpart(d2))
> > +map_localparts(f::Callable, d::DArray, ds::DArray...) = DArray(d) do I
> > +    f(localpart(d), map(localpart, ds)...)
>
> Isn't this covered by
> https://github.com/JuliaParallel/DistributedArrays.jl/blob/4e1a2804fcae0a4bde594f298cc3db484f894eca/src/DistributedArrays.jl#L965
> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaParallel/DistributedArrays.jl/pull/94/files/3737f8308114dffce3bc12360f785cfad98b0ffb#r73454180>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAI3anUIhznlP-ylDm0EUYvSeQMjJIzKks5qcU6egaJpZM4JcFxk>
> .
>